### PR TITLE
removed operation from persistence

### PR DESCRIPTION
**Pull request by StephenCoady**
##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: packages/offix-client/src/OfflineClient.ts [hunk 1]
```diff
@@ -135,13 +134,13 @@ export class OfflineClient implements ListenerProvider {
         variables: mutationOptions.variables
       })
 
+      // TODO we could probably move the conflictBase directly on to the mutationOptions.
+      // does createMutationOptions living in offix-cache make sense then though?
```
Comment (wtrocki) [inline packages/offix-client/src/OfflineClient.ts]: Generally, the side effects of the refactor to the scheduler are that people will not need to bother with Apollo cache. Apollo proposes to use graphql (and InMemoryCache) for a persisting state. I think people want to simply have state available (and persisted) - what I mean for us is that optimistic responses can become just optional feature that people can add (along with update methods) not the element on the critical path. 
Other offline providers use the same approach

Comment (wtrocki) [reply packages/offix-client/src/OfflineClient.ts]: TL;DR conflict base in mutationOptions will be against responsibility. Our base will be taken once someone will schedule offline operation

File: packages/offix-offline/src/offline/OperationQueueEntry.ts [hunk 2]
```diff
@@ -58,15 +64,17 @@ export class OperationQueueEntry implements OfflineItem {
...
 
   /**
    * Adapt object in order to persist required information
    */
   public toOfflineItem(): OfflineItem {
     return {
-      operation: this.operation,
+      query: this.query,
+      variables: this.variables,
+      operationName: this.operationName,
```
Comment (darahayes) [inline packages/offix-offline/src/offline/OperationQueueEntry.ts]: Question: do we/will we need the `context` in here?

**Comment by StephenCoady** [comment]
merged

**Pull request closed without merging**